Week4/MyFavoriteMovies/app/src/main/java/com/raywenderlich/myfavoritemovies/MovieDetailAdapter.kt

```diff
@@ -0,0 +1,24 @@
+package com.raywenderlich.myfavoritemovies
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+
+class MovieDetailAdapter(var movie: Movie?) : RecyclerView.Adapter<MovieDetailViewHolder>() {
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MovieDetailViewHolder {
+        val view = LayoutInflater.from(parent.context)
+            .inflate(R.layout.movie_detail_view_holder, parent, false)
+        return MovieDetailViewHolder(view)
+    }
+
+    override fun getItemCount(): Int {
+        return 1
```

Comment:
This doesn't really make sense to me. If the adapter is only ever going to have 1 item, why bother with a list/recyclerView, and an adapter? You can just use general views in the second activity, to achieve this behavior. That way it's easier to set it up, nicer to read through, and it's more intuitive!